src/proofPoints.ts

```diff
@@ -27,6 +27,42 @@ interface ProofPointIssueResult {
     proofPointObject: ProofPoint;
 };
 
+enum ProofPointValidity {
```

Comment:
Is there a way to define the values of enum in JS? Might be good in that case, bit like HTTP codes?

Reply:
Would `Status` be clearer than `Validity`? Whether valid or not is something inferred from different kinds of status perhaps?

Reply:
You can define the values yes. Are there particular values we want to give the items?

Reply:
Maybe this could be called status and then the ProofPointValidationResult would have a field `isValid` and a field `status`?